chart/values.yaml

```diff
@@ -254,6 +254,27 @@ apis:
   rest:
     # -- Log level for the rest service
     logLevel: info
+    healthProbes:
```

Comment:
Should we add an enable flag?

Reply:
We could. But should we? If the api-rest is not live, a restart should be the expected outcome?
Also, if the api-rest is not ready, it should be removed from the kubernetes service endpoint list?

Is there a reason to not want these? Should these be made mandatory, or should these be good-to-have features.

Reply:
If there's a bug, or something with these, we may want to disable them?

Reply:
Okay, I'll add toggles..

Reply:
I've added toggles